core/src/test/java/org/apache/iceberg/TestUpdateRequirements.java

```diff
@@ -198,11 +201,26 @@ public void upgradeFormatVersion() {
     assertTableUUID(requirements);
   }
 
+  @ParameterizedTest
+  @ValueSource(ints = {2, 3})
+  public void upgradeFormatVersionForView(int formatVersion) {
+    List<UpdateRequirement> requirements =
+        UpdateRequirements.forReplaceView(
+            viewMetadata, ImmutableList.of(new MetadataUpdate.UpgradeFormatVersion(formatVersion)));
+    requirements.forEach(req -> req.validate(viewMetadata));
+
+    assertThat(requirements)
+        .hasSize(1)
+        .hasOnlyElementsOfType(UpdateRequirement.AssertViewUUID.class);
+
+    assertViewUUID(requirements);
+  }
+
   @Test
-  public void upgradeFormatVersionForView() {
+  public void upgradeFormatVersionForViewV3() {
```

Comment:
see my comment above, we can revert these changes here

Reply:
Missed that one. Removing